packages/perseus/src/util/parse-perseus-json/perseus-parsers/grapher-widget.ts

```diff
@@ -42,58 +41,51 @@ export const parseGrapherWidget: Parser<GrapherWidget> = parseWidget(
                 "absolute_value",
                 object({
                     type: constant("absolute_value"),
-                    coords: pairOfPoints,
+                    coords: nullable(pairOfPoints),
                 }),
             )
             .withBranch(
                 "exponential",
                 object({
                     type: constant("exponential"),
                     asymptote: pairOfPoints,
-                    coords: pairOfPoints,
+                    coords: nullable(pairOfPoints),
                 }),
             )
             .withBranch(
                 "linear",
                 object({
                     type: constant("linear"),
-                    coords: defaulted(
-                        pairOfPoints,
-                        () =>
-                            [
-                                [-5, 5],
-                                [5, 5],
-                            ] as [[number, number], [number, number]],
-                    ),
```

Comment:
I previously thought we could default `coords` here, but thought better of it. This default isn't appropriate if the graph's X and Y ranges aren't the defaults ([-10, 10]).